src/rx5808-pro-diversity/settings.h

```diff
@@ -85,6 +84,13 @@ SOFTWARE.
 // Buzzer
 #define buzzer 6
 
+//activate if your buzzer is inverted compared to normal layout. 
+//#define BUZZER_INVERTED
+
+//aktivate to switch on a externel display with a short impulse on rxd
+//#define ACTIVATE_EXTERNAL_SCREEN
+#define RXD 0
```

Comment:
Everything is fine with your PR except i do not know what these lines are for.
90, 91, 92

If you remove them I'll merge the PR thanks.